master/getting-started/openshift/installation.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Installing Calico on OpenShift
+---
+
+Installation of Calico in OpenShift is integrated into the openshift-ansible scripts.
+
+## Basic
+
+For a basic Calico installation in OpenShift that shares the etcd
+instance used by the apiserver, set the following two `OSev3:vars` in your
+inventory file:
+
+  - `os_sdn_network_plugin_name=cni`
+  - `openshift_use_calico=true`
+
+Also ensure that you have an explicitly defined host in the `[etcd]` group.
+
+**Sample Inventory File:**
+
+```
+[OSEv3:children]
+masters
+nodes
+etcd
+
+[OSEv3:vars]
+os_sdn_network_plugin_name=cni
+openshift_use_calico=true
+
+[masters]
+master1
+
+[nodes]
+node1
+
+[etcd]
+etcd1
+```
+
+## Bring-your-own etcd
+
+Calico's OpenShift-ansible integration supports connection to a custom etcd which
+a user has set up.
+
+**Requirements:**
+
+  - The etcd instance must have SSL authentication enabled.
+  - Certs must be present at the specified filepath on all nodes.
+  - Certs must be in the same `calico_etcd_cert_dir`
+
+**Inventory Parameters:**
+
+| Key | Value     |
+| :------------- | :------------- |
+| `calico_etcd_endpoints` | Address of etcd. ex: `https://etcd:2379` |
+| `calico_etcd_ca_cert_file` | Absolute filepath of etcd CA cert. |
+| `calico_etcd_cert_file` | Absolute filepath of etcd CA file. |
+| `calico_etcd_key_file` | Absolute filepath of etcd cert. |
+| `calico_etcd_cert_dir` | Absolute filepath of etcd key file. |
+
+**Sample Inventory File:**
+
+```
+[OSEv3:children]
+masters
+nodes
+etcd
+
+[OSEv3:vars]
+os_sdn_network_plugin_name=cni
+openshift_use_calico=true
+calico_etcd_endpoints=http://etcd:2379
```

Comment:
Maybe rename it to calico_etcd to make it clear that its not the same etcd that is used by OpenShift?